ledger/acctonline_test.go

```diff
@@ -1484,7 +1485,20 @@ func TestAcctOnlineTop(t *testing.T) {
 
 func TestAcctOnlineTopInBatches(t *testing.T) {
 	partitiontest.PartitionTest(t)
-	a := require.New(t)
+
+	intToAddress := func(n int) basics.Address {
+		var addr basics.Address
+		pos := 0
+		for {
+			addr[pos] = byte(n % 10)
+			n /= 10
+			if n == 0 {
+				break
+			}
+			pos++
+		}
+		return addr
+	}
```

Comment:
nit: [we have something like this in the blockgen tests](https://github.com/algorand/go-algorand/blob/5e1f8a2077bc6ca2f181001f00a9c64903a609aa/tools/block-generator/generator/utils.go#LL51C1-L55C2)
```suggestion
	intToAddress := func(i uint64) (addr basics.Address) {
		// Make sure we don't generate a zero address by adding 1 to i
		binary.LittleEndian.PutUint64(addr[:], i+1)
		return
}
```

Reply:
this is different, one digit per byte, easier to debug